src/builder.rs

```diff
@@ -488,7 +493,8 @@ const fn get_texture_format_size(texture_format: wgpu::TextureFormat) -> f32 {
         | Rg8Snorm
         | Rg8Uint
         | Rg8Sint
-        | Rgb9e5Ufloat => 2.0, // 16.0 / 8.0
+        | Rgb9e5Ufloat
+        | Depth16Unorm => 2.0, // 16.0 / 8.0
```

Comment:
I have no idea if this is correct. "Moved" from https://github.com/parasyte/pixels/pull/312/files#diff-e4f794ca308aa712cbad31feb714df7cada1bbe453a30e232e45891571430263L514